crates/bevy_tasks/src/lib.rs

```diff
@@ -8,6 +8,37 @@
 
 extern crate alloc;
 
+#[cfg(not(target_arch = "wasm32"))]
+mod conditional_send {
+    /// Use [`ConditionalSend`] to mark an optional Send trait bound. Useful as on certain platforms (eg. Wasm),
+    /// futures aren't Send.
+    pub trait ConditionalSend: Send {}
+    impl<T: Send> ConditionalSend for T {}
+}
+
+#[cfg(target_arch = "wasm32")]
+#[expect(missing_docs, reason = "Not all docs are written yet (#3492).")]
+mod conditional_send {
+    pub trait ConditionalSend {}
+    impl<T> ConditionalSend for T {}
+}
+
+pub use conditional_send::*;
+
+/// Use [`ConditionalSendFuture`] for a future with an optional Send trait bound, as on certain platforms (eg. Wasm),
+/// futures aren't Send.
+pub trait ConditionalSendFuture: core::future::Future + ConditionalSend {}
+impl<T: core::future::Future + ConditionalSend> ConditionalSendFuture for T {}
+
+#[cfg(feature = "alloc")]
```

Comment:
It should be sufficient to just remove these `cfg(...)` attributes in order to remove the `alloc` feature. This will also need to be done in `futures.rs`.

Reply:
Thanks for chiming in on this one, I was unsure what to do with it.

Removed the feature and the cfgs. There were no instances of this in `futures.rs` FWIW.